style.css

```diff
@@ -781,14 +836,14 @@ ul {
     gap: 15%;
```

Comment:
Your implementation seems to have added a bar or the desktop version of the website. Kindly ensure that your correct it too so as to remain as given by the design.

![image](https://user-images.githubusercontent.com/20554583/233060675-0c1471ff-0aeb-468d-8905-254406614ba8.png)